ArduCopter/config.h

```diff
@@ -1,6 +1,8 @@
 //
 #pragma once
 
+#include <AP_ADSB/AP_ADSB.h>
```

Comment:
> why do we need this here ?

> why do we need this here ?

Well, I noted that we use `HAL_ADSB_ENABLED` in this file, so I thought, "sure, I can fix that".

*Big* mistake.  config.h requires all of Copter.h's includes before it gets included, and changing the order of things could be deadly...

This has now been reverted.

Reply:
> maybe just #include "AP_ADSB_config.h" ?

Yes, it should have been

Reply:
> why keep ADVANCED_FAILSAFE when you remove it from plane ?

So that we can build all of our boards with `AP_ADVANCEDFAILSAFE_ENABLED` set to true but Copter doesn't start to get it compiled in by default :-)